src/Raven.Server/ServerWide/Commands/CommandBase.cs

```diff
@@ -9,11 +9,12 @@ namespace Raven.Server.ServerWide.Commands
 {
     public abstract class CommandBase
     {
+        public string Type;
```

Comment:
ehm... I know what you are trying to do, but this will case a lot of confusion, each command will have this field now. Let's avoid it